R/grid.R

```diff
@@ -11,15 +11,84 @@
 #' gsNew <- points(gs, y=1, x=2, xlim=c(0,NA),ylim=c(0,NA),
 #'             col="blue", pch=18, legend.name="Points")
 #' gsNew <- lines(gsNew, c(3,4,3), c(2,4,6), legend.name="Lines")
-#' gsNew <- grid(gsNew) 
+#' gsNew <- grid(gsNew, legend.name="Grid") 
```

Comment:
OK, having the grid in the legend was in an example of ours a looong time ago. I also don't like the idea, so I'll quick take this out of the example, and merge the final pull request.